src/portals/hope/pages/tools/UnsignMemberPage.tsx

```diff
@@ -13,11 +12,20 @@ const UNSIGN_MEMBER = gql`
   }
 `
 
+const TERMINATE_CONTRACTS = gql`
+  mutation TerminateContracts($ssn: String, $email: String) {
```

Comment:
Codegen will autogenerate the hooks for you, so to avoid confusion maybe

```suggestion
  mutation TerminateStagingContracts($ssn: String, $email: String) {
```